civihr_default_theme/scss/pages/_hr-details.scss

```diff
@@ -65,3 +72,192 @@
+    margin-bottom: 0;
+  }
+
+  .views-row {
+    min-height: 80px;
+    padding-left: 0;
+    position: relative;
+    @media (min-width: 320px) {
+      padding-left: 100px;
+    }
+  }
+
+  .views-field-image-URL {
+    display: none;
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 80px;
+    @media (min-width: 320px) {
+      display: block;
+    }
+  }
+
+  .chr_panel__footer {
+    border-top: 0;
+    margin-top: 0;
+  }
+
+  .views-field-nothing {
+    .field-content {
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: space-between;
+      white-space: nowrap;
+      width: 100%;
+    }
+
+    .chr_panel--my-details__personal__contact_id,
+    .chr_panel--my-details__personal__gender,
+    .chr_panel--my-details__personal__marital_status {
+      margin-right: 10px;
+    }
+  }
+
+  .views-field-ni-ssn-70 .field-content {
+    width: 100%;
+  }
+
+  .chr_panel--my-details__personal__application_status--in-progress {
+    display: flex;
+
+    > * {
+      width: 50%;
+    }
+  }
+
+  .views-field.views-field-display-name {
+    color: $crm-gray-darker-alt;
+    font-size: $crm-font-size-large;
+    margin-bottom: 5px;
+
+    .views-label {
+      display: none;
+    }
+
+    > * {
+      width: 100%;
+    }
+  }
+
+  .chr_panel--my-details__personal__application_status {
+    > * {
+      display: inline-block;
+      float: left;
+      width: 50%;
+    }
+
+    &.not-applying,
+    &.No {
+      display: none;
+    }
+  }
+
+  .chr_actions-wrapper a {
+    margin: 10px 5px;
+  }
+}
+
+
+.view-mydetails-contactdetails {
+  .views-field-nick-name {
+    overflow: hidden;
+  }
+
+  .views-row {
+    margin-bottom: 15px;
+  }
+
+  .view {
+    clear: both;
+
+    .views-row {
+      display: flex;
+    }
+  }
+
+  .chr_panel__footer {
+    border-top: 0;
+    margin-top: 0;
+  }
+
+  .views-field-phone-ext {
+    order: 10;
+
+    &::before {
+      content: 'ext. ';
+      margin-left: 5px;
+    }
+  }
+}
+
+.view-mydetails-emails {
+  .views-field-email {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    width: 50%;
+  }
+}
+
+.view-mydetails-emails,
+.view-mydetails-myphones {
+  .views-field-location-type {
+    width: 50%;
+
+    .field-content {
+      @include views-label;
+    }
+  }
+}
+
+.view-emergency-contacts {
+  .views-label {
+    @include views-label;
+  }
+
+  .field-content {
+    p {
+      margin: 0;
+    }
+  }
+
+  .row {
+    margin-bottom: 15px;
+  }
+
+  .views-row {
+    border-bottom: 1px solid $gray-light;
+    margin-bottom: 10px;
+    padding: 10px 30px 10px 0;
+    position: relative;
+
+    &.views-row-last {
+      border-bottom: 0;
+      margin-bottom: 0;
+    }
+  }
+
+  .views-field-emergency-contact-deletion {
+    bottom: 10px;
+    font-size: $crm-font-size-large;
+    position: absolute;
+    right: 10px;
+    text-align: right;
+  }
+
+  .col-md-6:first-child {
+    .views-field {
+      margin-bottom: 15px;
+    }
+  }
+}
```

Comment:
Looks like you have not configured you IDE to add an empty line in the end. Please do that, and let us know if you need help.

Reply:
@deb1990 I already configured this on my IDE, don't know what happened. I also have integrated SCSS lint but maybe I overlooked certain warnings.

Reply:
This is not fixed.

Reply:
Fixed in next update.